sdk/textanalytics/Azure.AI.TextAnalytics/src/TextSentimentClass.cs

```diff
@@ -4,25 +4,30 @@
 namespace Azure.AI.TextAnalytics
 {
     /// <summary>
+    /// The predicted sentiment for a given span of text.
     /// </summary>
     #pragma warning disable CA1717 // Only FlagsAttribute enums should have plural names
     public enum TextSentimentClass
     #pragma warning restore
     {
         /// <summary>
+        /// Indicates that the text sentiment is positive.
         /// </summary>
         Positive,
 
         /// <summary>
+        /// Indicates that the text sentiment is neutral.
```

Comment:
There's no neutral sentiment. This infers the lack of sentiment.

Reply:
I can update this - would it make sense to call this None in that case?  That would be more consistent with .NET naming conventions.

Reply:
I prefer not to change the name here if possible since this is a very basic term for the service.